teacher.rb

```diff
@@ -0,0 +1,12 @@
+require '/person'
+
+class Teacher
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] OPTIONAL : It is better to choose one default value assignment style, either `=` or` : ` eg: 
`def initialize(age, specialization, name = "Unknown", parent_permission = true)`

I'm not sure but I think I got many errors because of this in my project because I did't know 😄

Reply:
Hello, @ShoiraTa I tried using "=" for parent_permission but rubocop doesn't allow it, it wants me to use keyword params instead.